src/bentoml/_internal/server/metrics/prometheus.py

```diff
@@ -23,12 +23,16 @@ def __init__(
         multiproc_dir: str | None = None,
     ):
         """
-        Set up multiproc_dir for prometheus to work in multiprocess mode,
-        which is required when working with Gunicorn server
+        PrometheusClient is BentoML's own prometheus client that extends the official Python client.
 
-        Warning: for this to work, prometheus_client library must be imported after
-        this function is called. It relies on the os.environ['PROMETHEUS_MULTIPROC_DIR']
-        to properly setup for multiprocess mode
+        It does set up multiprocess dir for Prometheus to work in multiprocess mode, which is required
+        for BentoML to work in production.
+
+        .. note::
+
+           For Prometheus to behave properly, ``prometheus_client`` must be imported after this client
+           is called. This has to do with ``prometheus_client`` relies on ``PROMEHEUS_MULTIPROC_DIR``, which
+           will be set by this client.
```

Comment:
```suggestion
           ``prometheus_client`` must be imported after this client is initialized. This is because  this client sets``PROMEHEUS_MULTIPROC_DIR``, which must be set before Prometheus is imported.
```